shell/src/main/java/alluxio/cli/fsadmin/doctor/ConfigurationCommand.java

```diff
@@ -78,11 +78,13 @@ public int run() throws IOException {
    */
   private void printInconsistentProperties(
       Map<Scope, List<InconsistentProperty>> inconsistentProperties) {
+    String readableNullValue = "no value set";
```

Comment:
I would just inline this, the name `readableNullValue` feels weird since the normal readable null value is the string "null"